### PR TITLE
Enable model picker label to show ellipsis instead of overflowing

### LINKED ISSUE #231369: Chat input toolbar buttons overflow container in narrow viewports
Image

### PR DESCRIPTION
**Pull request by roblourens**
Fix #231369

### CODE REVIEW

File: src/vs/workbench/contrib/chat/browser/media/chat.css [hunk 1]
```diff
@@ -694,6 +694,24 @@ have to be updated for changes to the rules above, or to support more deeply nes
 	margin-right: auto;
 }
 
+.interactive-session .chat-input-toolbars > .chat-execute-toolbar {
+	min-width: 0px;
+
+	.chat-modelPicker-item {
+		min-width: 0px;
```
Comment (justschen) [inline src/vs/workbench/contrib/chat/browser/media/chat.css:701]: won't this `min-width` and the `min-width` below be inherited from the parent?

Comment (justschen) [reply src/vs/workbench/contrib/chat/browser/media/chat.css:701]: ah you're right, I didn't see that above 👍

**Review by justschen — APPROVED** [review]
looks good, I think small nit is the structure (in this css file, i don't see a lot of nested css), but i don't think it really matters

```css
.interactive-session .chat-input-toolbars > .chat-execute-toolbar,
.interactive-session .chat-input-toolbars > .chat-execute-toolbar .chat-modelPicker-item {
	min-width: 0;
}

.interactive-session .chat-input-toolbars > .chat-execute-toolbar .chat-modelPicker-item .chat-model-label {
        min-width: 0;
	overflow: hidden;
	text-overflow: ellipsis;
}

.interactive-session .chat-input-toolbars > .chat-execute-toolbar .chat-modelPicker-item .codicon {
	flex-shrink: 0;
}
```

**Comment by roblourens** [comment]
Nested selectors are very new, but they are good and I think we can start to use them in more places.

**Pull request merged**